documents/description.md

```diff
@@ -1,9 +1,41 @@
 # Project description
 
 ## Motivation
+*Why is this project useful or interesting, what problem are you trying to address, and why is a DSL an appropriate solution?*
+
+My project is a personal wine tasting notes database. Essentially, users will be able to take brief but useful notes when tasting a wine and then compare their notes with those of wine critics. This is different from other wine tasting tools because it involves the creation of a user-friendly format for tasting notes and translates professional critic notes into this format. A DSL is an appropriate solution for several reasons. One semantic challenge of the project is the creation of a tasting notes language that is accessible to a wide range of users. The project will also pull wine reviews and tasting notes from databases and translate those note formats into my own format. This will allow users to perform side-by-side comparisons of their notes with reviews.
+
 
 ## Language domain
+*What is the domain that this language addresses, and why is the domain useful? Who will benefit from this language? Are there any other DSLs for this domain? If so, what are they, and how might they influence your language design and implementation?*
+
+The domain addressed by the language is anyone who wants to write wine tasting notes for themselves. Many "correct" standards for tasting notes have been created, but wine is a very personal experience that is different for everyone. People can benefit from the language by expressing wines in their own words and then comparing those to more traditional tasters. Since the professional tasting notes would be in the same format as the notes taken by the user, this would be an easy comparison. It would also greatly appeal to someone trying to learn about wine, since they would know whether their tasting notes were comparable to those of other tasters.
+
```

Comment:
It might be helpful to have examples for or links to these standards (distinct from the apps, linked below).